internal/releases/readme.go

```diff
@@ -0,0 +1,37 @@
+package releases
+
+import (
+	"regexp"
+	"strings"
+)
+
+// ciBadgeRegexp is used to find action in Github CI Badges
+var ciBadgeRegexp = regexp.MustCompile(`!\[.*\]\((?P<Action>https://github.com/.+)/badge.svg\)`)
+
+// charmBadgeRegexp is used to find a Charm's name in its CharmHub badge
+var charmBadgeRegexp = regexp.MustCompile(`!\[.*\]\(https://charmhub.io/(?P<Name>.+)/badge.svg\)`)
+
+// GetCiStages tries to extract CI stages from the Badges in the README
+func GetCiStages(readme string, repoName string) (badges []string) {
```

Comment:
Feels to me like this is a more appropriate name?

```suggestion
func GetCiBadges(readme string, repoName string) (badges []string) {
```

Reply:
It technically gets the link to the action, meaning it leaves out the `badge.svg` part of the URL; but I guess that semantically it makes more sense to go for `GetCiBadges` :)

Reply:
Seeing that it would turn the field `CiBadges` though makes me think maybe it's better to rename both to `GetCiActions` and `CiActions`, as that's more representative of what they are

Reply:
Yeh that makes sense too, thanks!